### ISSUE TITLE
A page about what the accessibility-ready tag for themes means

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#398]
- [x] I've confirmed that this document is not already in the [Docs Project](https://github.com/orgs/wpaccessibility/projects/3)

**Title**
The accessibility-ready program explained

**Description**
A page about what the accessibility-ready tag for themes means

**Reason for Adding**
We now only describe what the guidelines are and how to test for it, but I think we need more clarification about what accessibility-ready means, how to add the tag, that the theme will be audited and that the theme will be removed if you don't act after failing an audit.

**Additional context**
[Accessibility-ready Theme Reviews: Extending the Deadline](https://make.wordpress.org/accessibility/2026/07/23/accessibility-ready-theme-reviews-extending-the-deadline/)

Comment Joe: 
> This is because of how the theme directory works; tags are derived directly from the code of the theme, in the `tags` line of the theme’s header in `style.css`. As a result, only a code change to the theme can remove the theme from the tag listing.
We’ve wanted to be able to make changes to the theme directory so that showing the theme in the directory as accessibility-ready can be conditional on having the tag and *also* having post meta indicating approval, but haven’t been able to move that forward.
> So right now, this is the only available option.
